felix/ipsets/ipsets.go

```diff
@@ -767,11 +797,13 @@ func (s *IPSets) tryUpdates(ipsetFilter func(ipSetName string) bool, programmedI
 
 	// Ask each dirty IP set to write its updates to the stream.
 	var writeErr error
+	var touchedIPSets []string
 	for _, setName := range dirtyIPSets {
 		// Ask IP set to write its updates to the stream.
 		if log.IsLevelEnabled(log.DebugLevel) {
 			log.WithField("setName", setName).Debug("Writing updates to IP set.")
 		}
+		touchedIPSets = append(touchedIPSets, setName)
```

Comment:
shoudn't this be after the filter below? Or the filter below only means we are not interested in reporting back whether these IPs were programmed?

Reply:
The filter is only about the reporting.